actions/checkForAndPlaceOrder.ts

```diff
@@ -86,29 +103,49 @@ const _checkForAndPlaceOrder: ActionFn = async (
         chainContext.provider
       );
 
-      const { deleteConditionalOrder, error } = await _processConditionalOrder(
+      const pollResult = await _processConditionalOrder(
         owner,
-        network,
+        chainId,
         conditionalOrder,
         contract,
         multicall,
-        chainContext,
-        context
+        chainContext
       );
+      const error = pollResult !== undefined;
 
-      console.log(
-        `[checkForAndPlaceOrder] Check conditional order result: ${
-          error ? "❌" : "✅"
-        }`
-      );
+      // Specific handling for each error
+      if (pollResult) {
+        // Dont try again the same order
+        if (pollResult.result === PollResultCode.DONT_TRY_AGAIN) {
+          ordersPendingDelete.push(conditionalOrder);
+        }
 
-      hasErrors ||= error;
+        // TODO: Handle the other errors :) --> Store them in the registry and ignore blocks until the moment is right
+        //  TRY_ON_BLOCK
+        //  TRY_AT_EPOCH
+      }
 
-      if (deleteConditionalOrder) {
-        ordersPendingDelete.push(conditionalOrder);
+      // Log the result
+      const resultDescription =
+        pollResult !== undefined
```

Comment:
This check is done on top already.

```suggestion
        error
```